git-secrets

```diff
@@ -72,6 +93,19 @@ scan() {
   esac
 }
 
+scan_history_or_die() {
```

Comment:
Is there any way to DRY this up a bit? A large amount of content here was just copy/pasted from the existing method.